canonical-web-and-design/ubuntu.com
# Update landscape redirects

## Done

- Updated landscape redirects to comply with the following :
<img width="574" alt="Screenshot 2022-05-12 at 13 22 30" src="https://user-images.githubusercontent.com/17607612/168063782-a17fa069-f485-46c5-aa60-09d22d29af2f.png">


## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: 
- [ ]  https://ubuntu-com-11601.demos.haus/business/landscape/
- [ ]  https://ubuntu-com-11601.demos.haus/business/landscape/compliance
- [ ]  https://ubuntu-com-11601.demos.haus/business/landscape/how-to-get-landscape/
- [ ]  https://ubuntu-com-11601.demos.haus/business/landscape/systems-management/
- [ ]  https://ubuntu-com-11601.demos.haus/business/landscape/working-with-landscape/
- [ ]  https://ubuntu-com-11601.demos.haus/business/systems-management/features/
- [ ]  https://ubuntu-com-11601.demos.haus/business/systems-management/how-to-get-landscape/
- [ ]  https://ubuntu-com-11601.demos.haus/business/systems-management/working-with-landscape/
- [ ]  https://ubuntu-com-11601.demos.haus/category/product/landscape/
- [ ]  https://ubuntu-com-11601.demos.haus/management/compliance/
- [ ]  https://ubuntu-com-11601.demos.haus/management/features/
- [ ]  https://ubuntu-com-11601.demos.haus/management/how-to-get-landscape/
- [ ]  https://ubuntu-com-11601.demos.haus/management/how-to-get-landscape/contact-us/
- [ ]  https://ubuntu-com-11601.demos.haus/management/landscape/
- [ ]  https://ubuntu-com-11601.demos.haus/management/landscape-features/
- [ ]  https://ubuntu-com-11601.demos.haus/management/return-on-investment/
- [ ]  https://ubuntu-com-11601.demos.haus/management/working-with-landscape/

- Check changed paths against above screen shot
- Test all changed redirects and make sure all work as expected

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5297
